nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/FileInfo.java

```diff
@@ -217,6 +217,20 @@ public Builder group(String group) {
         }
     }
 
+    public static String permissionToString(int fileModeOctal) {
+        if (fileModeOctal > 0777 || fileModeOctal < 00) {
+            throw new RuntimeException("Invalid permission numerals");
```

Comment:
Sorry, I did not catch it earlier, but generic exception types (like `RuntimeException` or `Exception`) should not be thrown directly.
`IllegalArgumentException` would be appropriate for this case (don't forget to update the tests too).